docs/plans/2025-12-07-devtools-integration-analysis.md

```
@@ -0,0 +1,1878 @@
+```typescript
+const devtools = createDevtools({
+  output: 'console', // or 'file', 'stream'
+  format: 'pretty', // or 'json', 'compact'
+  filter: { atoms: true, flows: true, timing: true }
+})
+
+const scope = createScope({ extensions: [devtools] })
+```
+
+**Output:**
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)